src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -11,7 +11,7 @@
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
+    public static final String MESSAGE_SUCCESS = "ATAS has cleared all your tasks!";
```

Comment:
Shouldn't the message be `ATAS has deleted all students!` or something like that?

Reply:
oops LOL yeah thanks!